packages/legend-art/src/components/panel/PanelDropZone.tsx

```diff
@@ -49,17 +49,22 @@ export const PanelEntryDropZonePlaceholder: React.FC<{
   showPlaceholder: boolean;
   className?: string;
   label?: string;
+  borderless?: boolean;
 }> = (props) => {
-  const { children, showPlaceholder, className, label } = props;
+  const { children, showPlaceholder, className, borderless, label } = props;
```

Comment:
seems concerning to me, like always, is there a way to customize by allowing users to pass in some component for the placeholder they wish? if you don't like the placeholder, either use `showPlaceholder=false` or pass in that `placeholder` renderer, what do you think? We need to refactor a bit here though since now the relationship between `label`, `showPlaceholder` and `placeholder` seem messy

Reply:
passing in placeholder renderer with the label for that one, maybe makes it clearer? 🤔